prog/vnet/update_load_balancer.rb

```diff
@@ -18,7 +18,7 @@ def load_balancer
   end
 
   label def remove_load_balancer
-    vm.vm_host.sshable.cmd("sudo ip netns exec #{vm.inhost_name} nft --file -", stdin: generate_nat_rules(vm.ephemeral_net4.to_s, vm.nics.first.private_ipv4.network.to_s))
+    vm.vm_host.sshable.cmd("sudo ip netns exec #{vm.inhost_name} nft --file -", stdin: generate_nat_rules(vm.ephemeral_net4.to_s, vm.nics.first.private_ipv4.network.to_s)) if vm
```

Comment:
I'd  rewrite this with:

if vm
 ...
end

It's easy to miss that "if" at the end of that huuuuge line

Reply:
I removed the change all together, because I think #1828 is a better place to put this in.